feature/gnoi/system/tests/kill_a_daemon_test/README.md

```diff
@@ -0,0 +1,32 @@
+# gNOI-3.6: Kill a Daemon Test
+
+## Summary
+
+This is a negative test to kill some Software daemons and verify if the implementation is able to accurately stream the Health and all related artifacts for the subject daemon.
+  
+## Procedure
+
+* gNOI-3.6.1 : Kill configuration daemon.
+
+  Kill the process that manages device configuration using the **gNOI.KillProcessRequest_SIGNAL_ABRT** and restart flag set to False. This will terminate the process and will also dump a Core file, while maintaing the process in a down state. Verify if the leaf /components/component/healthz/state/status transitioned to **UNHEALTHY**.
+   1. If the software module has a status of **UNHEALTHY**, issue healthZ.Get() to collect more details on the event. Also, use HealthZ.Artifact() to collect artifacts like core dump, logs etc.
+   2. Initiate a **gNOI.KillProcessRequest_SIGNAL_HUP** operation to restart and recover the killed process.
```

Comment:
I don't think SIGNAL_HUP will start a process which has been stopped.  It doesn't look like gNOI has a feature to start a stopped process.  Probably you have to just work with SIGNAL_ABRT and restart=true.  See gNOI-3.5 which already does this (for a different process/bgp)